src/explore-education-statistics-frontend/src/modules/table-tool/TableToolPage.tsx

```diff
@@ -85,9 +87,18 @@ class TableToolPage extends Component<Props, State> {
     tableData: [],
   };
 
-  public static async getInitialProps() {
+  public static async getInitialProps({ query }: NextContext) {
     const themeMeta = await tableBuilderService.getThemes();
-    return { themeMeta };
+
+    const publication = themeMeta
+      .flatMap(option => option.topics)
```

Comment:
Will flatmap not cause a problem with IE11?

Reply:
Nope, it should be polyfilled correctly for IE11 (and Edge). This particular bit of code is also done server-side and also gets polyfilled for Node.

Reply:
Cool, just thought i would check